extensions/amp-facebook-like/1.0/base-element.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {FacebookBaseElement} from '../../amp-facebook/1.0/facebook-base-element';
+
+export class BaseElement extends FacebookBaseElement {}
+
+/** @override */
+BaseElement['props'] = {
+  ...FacebookBaseElement['props'],
+  'action': {attr: 'data-action'},
+  'colorScheme': {attr: 'data-colorscheme'},
```

Comment:
lowercase `colorscheme`? Or you can include this change in #34716

Reply:
I'll merge this and change in the linked PR, thanks!